db/migrate/20160115142023_remove_replicated_rows_from_newly_excluded_tables.rb

```diff
@@ -0,0 +1,25 @@
+class RemoveReplicatedRowsFromNewlyExcludedTables < ActiveRecord::Migration
+  class MiqEventDefinition < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ScanItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Configuration < ActiveRecord::Base
+    serialize :settings, Hash
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    region_cond = ActiveRecord::Base.region_to_conditions(File.read(File.join(Rails.root, "REGION")).to_i)
+    MiqEventDefinition.where.not(region_cond).delete_all
+    ScanItem.where.not(region_cond).delete_all
+
+    Configuration.where(:typ => "vmdb").each do |c|
```

Comment:
Will do. What is a "double serialized config" and what would I do to deal with it? Something like this?
```ruby
c = YAML.load(c) unless c.class == Hash
```
Wouldn't `serialize :settings, Hash` complain if it didn't deserialize to a Hash? Or is it something like the first value in the hash is a serialized string?